python/tk_multi_entity_app/loader_action_manager.py

```diff
@@ -99,11 +101,11 @@ def get_actions_for_publishes(self, sg_data_list, ui_area):
             ]
 
             # Bind all the action params to a single invocation of the _execute_hook.
-            a.triggered[()].connect(
-                lambda qt_action=a, actions=actions: self._execute_hook(
-                    qt_action, actions
-                )
+            handler = functools.partial(
+                self._execute_hook, qt_action=a, actions=actions
             )
+            a.triggered[()].connect(handler)
+
```

Comment:
No need for an extra variable.

```suggestion
            a.triggered[()].connect(
                functools.partial(
                    self._execute_hook, qt_action=a, actions=actions
                )
            )
```